.dockerignore

```diff
@@ -3,3 +3,26 @@ node_modules
 **/node_modules
 **/dist
 dist
+tmp
+**/tmp
```

Comment:
```suggestion
```
I think `tmp` already covers this

Reply:
I have tmp folders in the root and in sub-directories.

Reply:
Never mind, I read up on the matter.

Reply:
I think just `tmp` would also cover root and sub-directories, but `**/tmp` works as well

Reply:
From what I read, `tmp` only covers the root directory but `**/tmp` covers root and sub-directories. But I did find conflicting information.